moped-editor/src/utils/autocompleteHelpers.js

```diff
@@ -0,0 +1,14 @@
+/*
+/ MUI autocomplete filter function which limits number of options rendered in select menu
+*/
+export const filterOptions = (options, { inputValue, getOptionLabel }) => {
+  // limits options to ensure fast rendering
+  const limit = 40;
```

Comment:
This is totally fine; it works and limits the length of the dropdown. It does feel like burying a magic number in a file when it's really a configuration to the application that could be stored in [moped-editor/src/config.js](https://github.com/cityofaustin/atd-moped/blob/main/moped-editor/src/config.js), [moped-editor/constants/signalComponent.js](https://github.com/cityofaustin/atd-moped/tree/main/moped-editor/src/constants), or similar. None-the-less, it's working great - 👍

Reply:
thanks for abstracting this!